// Calling a function that doesn't exist should throw.
test();